tools/FakeItEasy.Build/Program.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using static Bullseye.Targets;
+using static SimpleExec.Command;
+
+namespace FakeItEasy.Build
+{
+    class Program
+    {
+        private const string TestsDirectory = "./artifacts/tests";
+        private const string Solution = "./FakeItEasy.sln";
+        private const string VersionInfoFile = "./src/VersionInfo.cs";
+        private const string RepoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+
+        private static readonly string[] ProjectsToPack =
+        {
+            "./src/FakeItEasy/FakeItEasy.csproj",
+            "./src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.CSharp.csproj",
+            "./src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.VisualBasic.csproj"
+        };
+
+        private const string AnalyzerMetaPackageNuspecPath = "./src/FakeItEasy.Analyzer.nuspec";
+
+        private static readonly string[] Pdbs =
+        {
+            "src/FakeItEasy/bin/Release/net40/FakeItEasy.pdb",
+            "src/FakeItEasy/bin/Release/netstandard1.6/FakeItEasy.pdb",
+            "src/FakeItEasy.Analyzer/bin/Release/FakeItEasy.Analyzer.Csharp.pdb",
+            "src/FakeItEasy.Analyzer/bin/Release/FakeItEasy.Analyzer.VisualBasic.pdb"
+        };
+
+        private static readonly IReadOnlyDictionary<string, string[]> TestSuites = new Dictionary<string, string[]>
+        {
+            ["unit"] = new[]
+            {
+                "tests/FakeItEasy.Tests",
+                "tests/FakeItEasy.Analyzer.CSharp.Tests",
+                "tests/FakeItEasy.Analyzer.VisualBasic.Tests",
+            },
+            ["integ"] = new[]
+            {
+                "tests/FakeItEasy.IntegrationTests",
+                "tests/FakeItEasy.IntegrationTests.VB",
+            },
+            ["spec"] = new[]
+            {
+                "tests/FakeItEasy.Specs"
+            },
+            ["approve"] = new[]
+            {
+                "tests/FakeItEasy.Tests.Approval"
+            }
+        };
+
+        private static string LogsDirectory = "./artifacts/logs";

```

Comment:
nit: `const` and up near `TestsDirectory`?

Reply:
Sure